cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,162 @@
+/// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('user clicks on Image content type', () => {
+    cy.get(`ul.admin-list span.label:contains('Image')`).then($el => {
+        const content= $el[1];
+        cy.get(content).parent().click({ force: true });
+    })
+});
+
+And('user types {string} into Caption text field', (value) => {
+    cy.getIframeBody('iframe.cke_wysiwyg_frame.cke_reset').find('p').type(value);
+})
+
+And('user uploads test image {string}', (fileName) => {
```

Comment:
I advise against using file names as conditionals - what happens If we replace one of the image files ?

you should rephrase it like 
And user uploads "main" image "main_image.jpg"
And user uploads "featured" image "feature_card_image.jpg"
And user uploads "thumbnail" image "thumbnail_image.jpg"
.....
so then you have 
And('user uploads {string} image {string}', (imageType, fileName) => {
if (imageType === "main"){
....
}
}